.travis.yml

```diff
@@ -58,47 +61,35 @@ env:
     # Enable parallel compilation e.g. for gRPC.
     # (The Travis CI worker is allowed to use up to 2 cores, but as of 07/2015 4 parallel compilations is actually faster.)
     - MAKEFLAGS=-j4
-    # Uncomment the next line to debug Python integration tests.
-    # NOTE: When -k is enabled, subsequent tests will fail because the test directory is not empty.
-    #       I.e. modify the Makefile to make sure that the test to be debugged is run first.
-    #- VT_TEST_FLAGS="-v -k"
     # Run go build and test with -p 4 (i.e. up to 4 packages are compiled/tested in parallel).
     # As of 07/2015 this value works best in a Travis CI container.
     - VT_GO_PARALLEL=4
+    - PHP_FLAGS="-d extension=$HOME/php/mongo.so"
+    - PATH="$HOME/php:$PATH"
+    - TEST_FLAGS="-docker=false -timeout 5m -follow -remote-stats http://enisoc.com:15123/travis/stats"
```

Comment:
I don't think it's a problem in the detection - it prints a message saying it timed out and will be killed. The problem is that the kill doesn't work. I had a similar issue with the php tests, where signals were not working at all to tear down the vtgateclienttest server. It seems to be a travis-specific thing.